### PR TITLE
Update attribute names

### PR DESCRIPTION
**Pull request by breedx-nr**
Resolves https://github.com/newrelic/dropwizard-metrics-newrelic/issues/8

This should keep us in sync with the changes in 
https://github.com/newrelic/dropwizard-metrics-newrelic/pull/17
and
https://github.com/newrelic/micrometer-registry-newrelic/pull/17

### CODE REVIEW

File: dropwizard/README.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ A DropWizard Timer is used to time things. Under the hood, it is implemented as
 Because this metric is a composite of other types, we model it as such, using the same kind of New Relic Metric types as above. 
 
 We model a Timer with a Count, 4 Gauges for the Meter, and 6 Gauges for the underlying Histogram.
-All of these have a `"name"` set to the name of the Timer, and `"sourceType" : "timer"`:
+All of these have a `"name"` set to the name of the Timer, and `"source.type" : "timer"`:
 * A Count metric, representing the total number of events that have been timed by the Timer, over its lifetime
 The value of the Count is the change in the number of events since the last reporting period happened.
 * 4 Gauge metrics, all of which have an attribute `"groupingAs": "rates"`
```
Comment (jkwatson) [inline dropwizard/README.md]: this `groupingAs` needs to go away.

File: dropwizard/README.md [hunk 2]
```diff
@@ -61,7 +61,7 @@ Second, it provides a set of average rates. One is the mean rate over the lifeti
 The others are the 1, 5 and 15 minute moving averages of the rates.
 
 We model this as a Count, and 4 Gauge metrics, all with the same `"name"`, which is set to the name of the Meter,
-and `"sourceType" : "meter"`:
+and `"source.type" : "meter"`:
 * A Count metric, representing the total number of events that have been seen by the meter, over its lifetime
 The value of the Count is the change in the number of samples since the last reporting period happened.
 * 4 Gauge metrics, all of which have an attribute `"groupingAs": "rates"`
```
Comment (jkwatson) [inline dropwizard/README.md]: get rid of the reference to the attribute

File: dropwizard/README.md [hunk 3]
```diff
@@ -34,18 +34,18 @@ Because it can be decremented, we cannot model this as a New Relic Count metric,
...
 Since New Relic does not currently have a native percentile type, we instead have to model this as a set of other metrics.
 
 A DropWizard Histogram is converted to the following, all with the same `"name"`, which is set to the name of the Histogram,
-and `"sourceType" : "histogram"`:
+and `"source.type" : "histogram"`:
 * A Count metric, representing the total number of samples that have been added to the Histogram since its creation. 
 The value of the Count is the change in the number of samples since the last reporting period happened.
 * A set of Gauge metrics, one per standard percentile: 50, 75, 90, 95, 99, 99.9
-  * On each of these, we add an attribute `"percentile"` with the decimal value of what is being measured (eg. 99.9)
+  * On each of these, we add an attribute `"newRelic.percentile"` with the decimal value of what is being measured (eg. 99.9)
```
[the remark this answers is not in the document]
Comment (breedx-nr) [reply dropwizard/README.md]: indeed

File: dropwizard/README.md [hunk 4]
```diff
@@ -85,6 +85,6 @@ The value of the Count is the change in the number of events since the last repo
   * The 5-minute moving average rate, with attribute `"rate": "m5_rate"`
   * The 15-minute moving average rate, with attribute `"rate": "m15_rate"`
 * A Set of Gauge metrics, one per standard percentile: 50, 75, 90, 95, 99, 99.9, all of which have an attribute `"groupingAs": "percentiles"`
-  * On each of these, we add an attribute `"percentile"` with the decimal value of what is being measured (eg. 99.9)
+  * On each of these, we add an attribute `"newRelic.percentile"` with the decimal value of what is being measured (eg. 99.9)
```
[the remark this answers is not in the document]
Comment (breedx-nr) [reply dropwizard/README.md]: indeed

File: dropwizard/README.md [hunk 5]
```diff
@@ -85,6 +85,6 @@ The value of the Count is the change in the number of events since the last repo
   * The 5-minute moving average rate, with attribute `"rate": "m5_rate"`
   * The 15-minute moving average rate, with attribute `"rate": "m15_rate"`
```
[the remark this answers is not in the document]
Comment (jkwatson) [reply dropwizard/README.md]: we can decide to change this sometime in the future. not part of this PR

**Pull request merged**